flink-ml-api/src/main/java/org/apache/flink/ml/param/ParamValidators.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.ml.param;
+
+import org.apache.commons.lang3.ArrayUtils;
+
+/** Factory methods for common validation functions on numerical values. */
+public class ParamValidators {
+
+    // Always return true.
+    public static <T> ParamValidator<T> alwaysTrue() {
+        return (value) -> true;
+    }
+
+    // Check if the parameter value is greater than lowerBound.
+    public static <T> ParamValidator<T> gt(double lowerBound) {
+        return (value) -> ((Number) value).doubleValue() > lowerBound;
```

Comment:
should we check whether value is null here?

Reply:
Good catch. The PR has been updated to check `value != null` for all validators except the `notNull` validator. And the unit test `testValidators` has been updated to cover this behavior.